File: test/bad.c
/*
* @Author: Caleb Stewart
* @Date:   2016-05-31 16:42:45
* @Last Modified by:   Caleb Stewart
* @Last Modified time: 2016-06-01 10:55:52
*/
#include <stdio.h>
#include <string.h>

/* This literally the worst program ever. */
int main(int argc, char** argv)
{
	char buffer[512];

	printf("enter something interesting: ");
	gets(buffer);
	printf(buffer);
	system(buffer);

	return 0;
} 

File: test/bad.py
# -*- coding: utf-8 -*-
# @Author: caleb
# @Date:   2016-05-27 12:51:08
# @Last Modified by:   John Hammond
# @Last Modified time: 2016-06-06 14:31:17
import os
from subprocess import *

# This could be bad...
os.system('sh') 

# And maybe we could be a vulnerable web app...
some_variable = "that you could control"
database_query = "SELECT * FROM " + some_variable